pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -362,6 +362,11 @@ stdenv.mkDerivation {
     ''
 
     + ''
+      # There are a few tools (to name one libstdcxx5) which do not work
+      # well with multi line flags, so make the flags single line again
+      tr '\n' ' ' < $out/nix-support/libc-cflags > $out/nix-support/libc-cflags-one
+      mv $out/nix-support/libc-cflags-one $out/nix-support/libc-cflags
```

Comment:
OK. This can also be done without mv: `substituteInPlace $out/nix-support/libc-cflags --replace $'\n' ' '`.

Reply:
I missed this, its the nicer solution